src/parser/ytmusic/Playlist.ts

```diff
@@ -60,22 +59,22 @@ class Playlist {
     let section_continuation = this.#page.contents_memo.get('SectionList')?.[0].as(SectionList).continuation;
 
     while (section_continuation) {
-        const response = await this.#actions.browse(section_continuation, { is_ctoken: true, client: 'YTMUSIC' });
-        const data = Parser.parseResponse(response.data);
-        const section_list = data.continuation_contents?.as(SectionListContinuation);
-        const sections = section_list?.contents?.as(MusicCarouselShelf);
-        const related = sections?.filter(section => section.header?.title === 'Related playlists')[0];
-        if (related) {
-            return related.contents || [];
-        }
-        else {
-            section_continuation = section_list?.continuation;
-        }
+      const response = await this.#actions.browse(section_continuation, { is_ctoken: true, client: 'YTMUSIC' });
+      const data = Parser.parseResponse(response.data);
+      const section_list = data.continuation_contents?.as(SectionListContinuation);
+      const sections = section_list?.contents?.as(MusicCarouselShelf);
+      const related = sections?.filter((section) => section.header?.title === 'Related playlists')[0];
```

Comment:
Replace this line with `const related = sections?.filter((section) => section.header?.title?.toString() === 'Related playlists')[0];` to fix the error message.

Reply:
> Replace this line with `const related = sections?.filter((section) => section.header?.title?.toString() === 'Related playlists')[0];` to fix the error message.


Hmm.
The weird thing though is that `MusicCarouselShelf`'s header is of type `MusicCarouselShelfBasicHeader`. And that header's title is not a `Text`.

https://github.com/LuanRT/YouTube.js/blob/050addbcccd3aaaec5bfc4bf3474496efdaa3d83/src/parser/classes/MusicCarouselShelfBasicHeader.js#L15

Something's wrong here.

Reply:
Yes, something funky is happening in the Github Actions, since it does compile fine when I pull it locally.

Reply:
Same here, I'm gonna just merge this then.